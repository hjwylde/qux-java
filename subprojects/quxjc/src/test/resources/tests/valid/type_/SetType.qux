package valid.type_

type A is {int}
type B is {any}

{any} f(A a, B b):
    return a + b

void main([str] args):
    a = {5, 4}
    b = {2, 3}
    c = {2, 3, null}
    print f(a, a)
    print f(a, b)
    print f(b, c)
    print f(b, b)

