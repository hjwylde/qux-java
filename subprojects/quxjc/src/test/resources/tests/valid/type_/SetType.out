{4, 5}
{2, 3, 4, 5}
{2, 3, null}
{2, 3}
